packages/headless/src/features/commerce/common/actions.ts

```diff
@@ -41,6 +46,11 @@ export interface QueryCommerceAPIThunkReturn {
   analyticsAction: PreparableAnalyticsAction<StateNeededByQueryCommerceAPI>;
 }
 
+export interface QueryRecommendationCommerceAPIThunkReturn {
+  /** The successful search response. */
```

Comment:
comment + this should be moved to `recommendations-actions.ts`
```suggestion
  /** The successful recommendations response. */
```